lib/dpdk/dpdkCore.cpp

```diff
@@ -310,7 +310,7 @@ int dpdkCore::lcore_rx(void* args) {
         if (core->handlers[port] == nullptr) {
             // This is the one place (static member function) where normal logging does work.
             fprintf(stderr, "No valid handler provided for port %d", port);
-            raise(SIGINT);
+            FATAL_ERROR("No valid handler provided for port %d", port);
```

Comment:
Ah this may not work.  The lcore thread is another special case.  Basically for a somewhat complex reason the logging doesn't quite work as you'd expect here.  Which is why it uses `fprintf(stderr, ...);` above.   Generally I think you should just ignore cases were the error messages are printed with `printf` and leave them as they are now with the `raise()`.